### PR TITLE
Update civicrm_state_province.tpl

### PR DESCRIPTION
**Pull request by ghost**
Overview
----------------------------------------
I live in Norway, where 2 counties combined and underwent a name change to Vestland.
You can find more information about Vestland on [Wikipedia](https://en.wikipedia.org/wiki/Vestland).
Vestland was created in 2020 when the former counties of Hordaland and Sogn og Fjordane were merged.

Before
----------------------------------------
Sogn og Fjordane and Hordaland.

After
----------------------------------------
Sogn og Fjordane & Hordaland, added Vestland

Technical Details
----------------------------------------
Please review the changes and check for any mistakes, this is my first ever pull request.

Comments
----------------------------------------
Have a nice day! <3

### CODE REVIEW

File: xml/templates/civicrm_state_province.tpl [hunk 1]
```diff
@@ -2386,19 +2386,18 @@ INSERT INTO civicrm_state_province (id, country_id, abbreviation, name) VALUES
...
-(3959, 1161, "06", "Telemark"),
-(3960, 1161, "19", "Troms"),
-(3961, 1161, "10", "Vest-Agder"),
-(3962, 1161, "07", "Vestfold"),
+(3957, 1161, "16", "Sør-Trøndelag"),
+(3958, 1161, "06", "Telemark"),
+(3959, 1161, "19", "Troms"),
+(3960, 1161, "10", "Vest-Agder"),
+(3961, 1161, "07", "Vestfold"),
+(3962. 1161, "46", "Vestland")
```
Comment (colemanw) [inline xml/templates/civicrm_state_province.tpl:2400]: @midtsveen this punctuation will cause a fatal syntax error

Comment (midtsveen) [reply xml/templates/civicrm_state_province.tpl]: Updated

**Comment by colemanw** [comment]
Ok thanks @midtsveen looks good now.

@civicrm-builder add to whitelist.

**Comment by midtsveen** [comment]
I want a minimal repo list, and I can delete my own forked version of this repository without any problems once it is merged, right?

File: xml/templates/civicrm_state_province.tpl [hunk 2]
```diff
@@ -2386,19 +2386,18 @@ INSERT INTO civicrm_state_province (id, country_id, abbreviation, name) VALUES
...
-(3959, 1161, "06", "Telemark"),
-(3960, 1161, "19", "Troms"),
-(3961, 1161, "10", "Vest-Agder"),
-(3962, 1161, "07", "Vestfold"),
+(3957, 1161, "16", "Sør-Trøndelag"),
+(3958, 1161, "06", "Telemark"),
+(3959, 1161, "19", "Troms"),
+(3960, 1161, "10", "Vest-Agder"),
+(3961, 1161, "07", "Vestfold"),
+(3962, 1161, "46", "Vestland")
```
Comment (eileenmcnaughton) [inline xml/templates/civicrm_state_province.tpl:2400]: there is a comma missing at the end of this line - hence the test fails

**Comment by eileenmcnaughton** [comment]
@midtsveen I'm going to merge this - but you should be aware that changing this file will only affect new installs - there needs to be something in the upgrade script to apply changes to existing sites (I think we normally add & rename but not delete for existing sites since the old ones might be in use)


eg. https://github.com/civicrm/civicrm-core/blob/68b851fea4d4f12c49dda42b2858f90125f0ca19/CRM/Upgrade/Incremental/sql/5.59.alpha1.mysql.tpl#L4

**Pull request merged**